src/Files.Core/Services/IAppThemeModeService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Core.Services
+{
+	public interface IAppThemeModeService
```

Comment:
Do you think we can simplify this to `IThemeService`?

Reply:
There's already AppThemeBackground and App would be necessary for such application, which have a lot of features to differentiate others.

![image](https://github.com/files-community/Files/assets/62196528/a799cd20-fe9d-40ab-b625-143ee0f35197)

Reply:
This is still on planning stage but I'm planning rename ResourcesService, JumpListService, UpdateService to have App prefix, move into Files.App.Services.Application.*.

![image](https://github.com/files-community/Files/assets/62196528/114c89b4-5e1a-4bc4-baab-8d13325a00dc)

This is also on planning stage but I'm planning to rename those 2 helpers (DragZoneHelper & JumpListHelper) to have App prefix to be consistent
![image](https://github.com/files-community/Files/assets/62196528/b679380f-5244-4b82-94d5-99b2eb7e5a38)

I have some outlook for the future of improvements on Files codebase, so I'd like to keep as it is.